src/deluge/util/chainload.h

```diff
@@ -5,4 +5,7 @@
 #define OFF_USER_CODE_EXECUTE 0x28
 #define OFF_USER_SIGNATURE 0x2c
 
+/// Chainload from a buffer.
+///
+/// Currently only implemented for ARM.
```

Comment:
can we add a note about entry point location being the value of buffer+0x28 for the sake of future us?